unstable/wlr-workspace-unstable-v1.xml

```diff
@@ -0,0 +1,192 @@
+      outputs, such that activating the workspace will request that the compositor
+      display the workspace's surfaces on those outputs, whereas the compositor may hide
+      or otherwise de-emphasise surfaces that are associated only with inactive
+      workspaces. By associating a set of outputs with each workspace, each output may
+      have its own unique set of workspaces, or all outputs (or any other arbitrary
+      grouping) may share workspaces. Workspaces may be conceptually arranged in a grid
+      of arbitrary dimension. The purpose of this protocol is to enable the creation of
+      taskbars and docks by providing them with a list of workspaces and their
+      properties, and allowing them to activate and deactivate workspaces.
+
+      After a client binds the zwlr_workspace_manager_v1, each workspace will be sent
+      via the workspace event.
+    </description>
+
+    <event name="workspace">
+      <description summary="a workspace has been created">
+        This event is emitted whenever a new workspace has been created.
+
+        All initial details of the workspace (outputs, name, coordinates) will be sent
+        immediately after this event via the corresponding events in
+        zwlr_workspace_handle_v1.
+      </description>
+      <arg name="workspace" type="new_id" interface="zwlr_workspace_handle_v1"/>
+    </event>
+
+    <event name="finished">
+      <description summary="the compositor has finished with the workspace_manager">
+        This event indicates that the compositor is done sending events to the
+        zwlr_workspace_manager_v1. The server will destroy the object immediately after
+        sending this request, so it will become invalid and the client should free any
+        resources associated with it.
+      </description> 
+    </event> 
+
+    <request name="stop">
+      <description summary="stop sending events">
+        Indicates the client no longer wishes to receive events for new workspaces.
+        However the compositor may emit further workspace events, until the finished
+        event is emitted.
+
+        The client must not send any more requests after this one.
+      </description>
+    </request>
+  </interface>
+
+  <interface name="zwlr_workspace_handle_v1" version="1">
+    <description summary="a workspace handing a group of surfaces">
+      A zwlr_workspace_handle_v1 object represents a a workspace that handles a group of
+      surfaces.
+
+      Each workspace has a name, conveyed to the client with the name event; a list of
+      states, conveyed to the client with the state event; a list of outputs, conveyed
+      via one or more output events; and optionally a set of coordinates, conveyed to
+      the client with the coordinates event. The client may request that the compositor
+      activate or deactivate the workspace.
+    </description>
+
+    <event name="output">
```

Comment:
I'd name this `enter` (and rename `output_remove` to `leave`) for consistency with the rest of the Wayland protocol.